src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(resultSoFar, number) {
+      return resultSoFar + number;
+    },
```

Comment:
this should take only number as an argument and assign the result to the `result` variable